pupil-spa/src/app/services/pupil-prefs-sync/pupil-prefs-sync.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+
+import { AccessArrangements, AccessArrangementsConfig, accessArrangementsDataKey } from '../../access-arrangements';
+import { StorageService } from '../storage/storage.service';
+
+@Injectable()
+export class PupilPrefsSyncService {
+  accessArrangements;
+  fontSettings;
+  contrastSettings;
+
+  constructor(
+    private storageService: StorageService,
+  ) {}
+  /*
+  // Fetch pref values by first at access_arrangements key and then at config key
+  // If both are not supplied provide default prefs
+   */
+  public sync() {
```

Comment:
Why is this service called `sync`?  It appears to be loading config.

Reply:
@jon-shipley refactored.